internal/tools/aws/rds.go

```diff
@@ -93,12 +93,13 @@ func (a *Client) rdsEnsureDBClusterCreated(awsID, vpcID, username, password, kms
 		return err
 	}
 
+	azs, err := a.getAvailabilityZones()
+	if err != nil {
+		return err
+	}
+
 	input := &rds.CreateDBClusterInput{
-		AvailabilityZones: []*string{
-			aws.String("us-east-1a"),
-			aws.String("us-east-1b"),
-			aws.String("us-east-1c"),
-		},
+		AvailabilityZones:     azs,
```

Comment:
Here, I dont know if we want to cap to 2-3 or want to add all available

Reply:
This should be fine, I would think.